lightningd/opening_control.c

```diff
@@ -1167,6 +1161,15 @@ static struct command_result *json_fundchannel_start(struct command *cmd,
 	peer->uncommitted_channel->fc = tal_steal(peer->uncommitted_channel, fc);
 	fc->uc = peer->uncommitted_channel;
 
+	/* BOLT #2:
+	 *
+	 * The sender:
+	 *   - SHOULD set `minimum_depth` to a number of blocks it considers
+	 *     reasonable to avoid double-spending of the funding transaction.
+	 */
+	assert(mindepth != NULL);
```

Comment:
I usually skip assert here, and just rely on the next line crashing :)

Reply:
Mhe, I find the assert clearer to root-cause (one less indirection) but can change it either way.